tutorials/2_sprite_animations_gestures/README.md

```diff
@@ -0,0 +1,241 @@
+# Basic: Sprites, Animations and Gestures
+
+This tutorial will introduce you to:
+
+ - `Sprite`: Sprites are how we draw images, or portions of an image in Flame.
+ - `SpriteAnimation`: SpriteAnimations are animations composed from sprites, where each sprite represents a frame.
+ - Gesture input: Input is a crucial part of game development, on this tutorial you will learn how to use one of the most basic of them, the tap detector.
```

Comment:
I think we don't need to say `on this tutorial you will learn` again, that is already the title for this list
```suggestion
 - Gesture input: the most basic input for your game, the tap detector.
```
not sure if my suggestion encompasses everything on the original version but feels weird to repeat the "this tutorial will teach" bit